db/migrate/20200211105449_remove_unused_person_id_from_move.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveUnusedPersonIdFromMove < ActiveRecord::Migration[5.2]
+  def up
+    remove_column :moves, :person_id
+  end
+  def down
+    change_table :moves do |t|
+      t.uuid :person_id
+      t.index ["from_location_id", "to_location_id", "person_id", "date"], name: "index_on_move_uniqueness", unique: true
```

Comment:
Why do you add index to 
from_location_id, to_location_id and date
hen you rollback the migration?